Sources/Telemetry/Telemeter/Telemeter.swift

```diff
@@ -21,12 +21,14 @@
 // SOFTWARE.
 
 import Foundation
-import os
 
 /// A type that can record ``Metron``s.
 public protocol Telemeter {
+
     /// Records a ``Metron`` along with the provided ``Facet``s to the receiving ``Telemeter``.
     ///
+    /// This function is intended only for internal Telemetry use. Users should prefer ``record(_:file:line:)`` when recording Metrons so that the File facet is appened.
+    ///
```

Comment:
@davidbarry-kr added this per your comment.

Reply:
Looks good!